### PR TITLE
Michelle/dbtables

### PR DESCRIPTION
**Pull request by michellemalf**
init_db: create tables for users, products, orders, order_products

### CODE REVIEW

**Review by richardedwardbrown3 — APPROVED** [review]
Looks good!

**Review by Felix-Cadiz — APPROVED** [review]
LGTM! I'll make small changes to formatting and users.js. Some of the values are only listed to two when we need more parameters.

File: db/init_db.js [hunk 1]
```diff
@@ -1,21 +1,71 @@
...
+        DROP TABLE IF EXISTS users, products, orders, order_products;
+      `);
+  
+      await client.query(`
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY, 
+        firstname VARCHAR(255) NOT NULL, 
+        lastname VARCHAR(255) NOT NULL, 
+        email VARCHAR(255) UNIQUE NOT NULL, 
+        "imgURL" DEFAULT VALUE,
```
Comment (kbonifacio) [inline db/init_db.js:20]: Update on DEFAULT constraint. The word value should be changed to what we actually want as the default value (e.g. if we want some default avatar).

File: db/init_db.js [hunk 2]
```diff
@@ -1,21 +1,71 @@
...
+        address NOT NULL
+      );
+    `);
+    await client.query(`
+    CREATE TABLE products(
+      id SERIAL PRIMARY KEY, 
+      name VARCHAR(255) UNIQUE NOT NULL, 
+      description STRING, 
+      price INTEGER NOT NULL, 
+      "imgURL" DEFAULT VALUE,
```
Comment (kbonifacio) [inline db/init_db.js:33]: Update on DEFAULT constraint. The word value should be changed to what we actually want as the default value (e.g. if we want some default avatar).

**Review by kbonifacio — APPROVED** [review]
lgtm

**Comment by kbonifacio** [comment]
Created dbtables and initial queries

**Comment by kbonifacio** [comment]
created dbtables and initial queries; will need to update default values for "imgURL" and a later point in time.

**Pull request merged**